services/web-app/components/sort-by-dropdown/sort-by-dropdown.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
```

Comment:
```suggestion
 * Copyright IBM Corp. 2022, 2022
```

This looks like a net-new file. If it's a refactor of existing stuff, we can keep the date at 2021.

Reply:
this is mostly copied over from `services/web-app/components/catalog-sort/catalog-sort.js`, should we keep it at 2021 then? or change it? 🤔

Reply:
Okay, np. We can keep it as 2021